services/app-api/src/postgres/contractAndRates/approveContract.ts

```diff
@@ -56,7 +56,7 @@ async function approveContractInsideTransaction(
 type ApproveContractArgsType = {
     contractID: string
     updatedByID: string
-    updatedReason: string | undefined
+    dateApprovalReleasedToState: string | Date
```

Comment:
**question**: Wondering why this would be either string or date? We would know what the input type `dateApprovalReleasedToState ` should be based on the GraphQL schema, which is a `Date`.